project.clj

```diff
@@ -16,7 +16,10 @@
                  [com.datomic/datomic-free "0.9.5130" :exclusions [joda-time]]
                  [clj-http "1.1.2"]
                  [raven-clj "1.1.0"]
-                 [io.framed/std "0.1.2"]]
+                 [io.framed/std "0.1.2"]
+                 [aysylu/loom "1.0.0"]
+                 [org.clojure/java.jdbc "0.7.0-alpha3"]
+                 [com.h2database/h2 "1.4.195"]]
```

Comment:
I'm assuming H2 for development and test environments.  This alleviates the need to have an external MySQL instance running in order to do things like run tests.